src/PasswordResetController.js

```diff
@@ -91,6 +90,10 @@ function (Okta, FormController, FormType, ValidationUtil, FactorUtil, FooterSign
         this.settings.processCreds(creds)
           .then(_.bind(this.model.save, this.model));
       });
+
+      if (!this.settings.get('features.hideBackToSignInForReset')) {
+        this.addFooter(FooterSignout, {linkText: Okta.loc('goback', 'login'), linkClassName: ''});
```

Comment:
question: did we agree on changing the button text here to `Back`? This controller is for when the user is changing the password, so the user is already in the last step... just want to confirm, this could make sense to leave it as it was with "Sign Out"

Reply:
Yes, we thought about it and it's in the design doc. @vincentvoong-okta also agreed it would make sense to change to "Back to Sign In", which would be consistent with the rest of the flow.
However, what you're saying is true. I think at this point, the user is authenticated. So we could say "Sign Out" instead of "Back to Sign In".  I don't really have a strong opinion, because the meaning of both these labels is very similar, and they both go to the primary auth page.
Maybe we can leave this to say "Sign Out" for now.